guide/src/examples/wasm-in-web-worker.md

```diff
@@ -0,0 +1,71 @@
+# WASM in Web Worker
+
+[View full source code][code]
```

Comment:
I saw that other examples not only point to the code but also to a compiled demo - but that is not autogenerated for each example, right?